src/content/en/2025/cookies.md

```diff
@@ -3,18 +3,646 @@
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Statistics for age of cookies set on desktop pages.") }}</figcaption>
+</figure>
+
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Age of cookies (mobile top 1M) in days </th>
+        <th>First-party</th>
+        <th>Third-party</th>
+        <th>All</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>min</td>
+        <td class="numeric">0</td>
+        <td class="numeric">0</td>
+        <td class="numeric">0</td>
+      </tr>
+      <tr>
+        <td>p25</td>
+        <td class="numeric">1</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>median</td>
+        <td class="numeric">365</td>
+        <td class="numeric">270</td>
+        <td class="numeric">360</td>
+      </tr>
+      <tr>
+        <td>p75</td>
+        <td class="numeric">395</td>
+        <td class="numeric">365</td>
+        <td class="numeric">390</td>
+      </tr>
+      <tr>
+        <td>p90</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+      <tr>
+        <td>p99</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+      <tr>
+        <td>max</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+        <td class="numeric">400</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Statistics for age of cookies set on mobile pages.") }}</figcaption>
+</figure>
+
+Cookies are set to an expiration date when they are created. If session cookies expire immediately after the session is over ([see previous section](#session)), most first- and third-party cookies do not and have a median age of a full year. The longer cookies live, the longer they can be used for re-identification or cross-site tracking which is why most tracking cookies are typically set to be stored in the browser for a longer time. The maximum age among the cookies that we can observe with the instrumentation and collection of the HTTP Archive Tools is of 400 days, this is aligned with the [hard limits](https://developer.chrome.com/blog/cookie-max-age-expires) that Chrome imposes on cookie `Expires` and `Max-Age` attribute.
+
+{# TODO: if any other idea of what to say here, feel free to add #}
+
+{# TODO (or idea for future): check the issue tab data of Chrome devtools and/or the console warnings related to cookies, data is normally collected in HTTP Archive. There could be a dedicated section in this chapter about it #}
+
+## Conclusion
+
+The observations from this chapter confirm the conclusions from last year's analysis:
+- A majority (60%) of cookies encountered on the web are third-party cookies and popular websites create them the most.
```

Comment:
1. "popular websites create them the most.", the cookies are created by the third-party (okay thee third-parties are most of the time also popular sites) and not the site visited. Maybe "popular websites have significantly more third-party cookies than less popular sites."